recipes-bsp/u-boot/files/0013-board-iot2050-Reset-the-eMMC-when-it-isn-t-reachable.patch

```diff
@@ -0,0 +1,129 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Li Hua Qian <huaqian.li@siemens.com>
+Date: Wed, 16 Oct 2024 17:03:10 +0800
+Subject: [PATCH] board: iot2050: Reset the eMMC when it isn't reachable
+
+After approximately 2 months of operation, the device may fail to detect
+the eMMC when rebooting. This commit provides a temporary workaround to
+reset the eMMC if detection issues occur.
+
```

Comment:
Is this issue likely related to how the board is wiring the eMMC, or is this a generic problem that should be addressed at eMMC driver level in U-Boot?

Reply:
"For backward compatibility reason, RST_n signal is temporary disabled in device by default."  - eMMC standard.

Therefore, this feature is not mandatory. If the workaround proves effective, we'll consider incorporating it into the eMMC driver.